### PR TITLE
Minor README tweaks

### PR DESCRIPTION
**Pull request by Zearin**
formatting, phrasing, etc.

### CODE REVIEW

**Comment by samccone** [comment]
@thejameskyle mind taking a look?

File: README.md [hunk 1]
```diff
@@ -19,23 +19,23 @@
...
-* Deterministic: The same dependencies will be installed the same exact way across every machine regardless of install order.
-* Network Performance: Yarn efficiently queues up requests and avoids request waterfalls in order to maximize network utilization.
-* Multiple Registries: Install any package from either npm or Bower and keep your package workflow the same.
-* Network Resilience: A single request failing won't cause an install to fail. Requests are retried upon failure.
-* Flat Mode: Resolve mismatching versions of dependencies to a single version to avoid creating duplicates.
-* More emojis. 🐈
+* **Offline Mode.** If you've installed a package before, you can install it again without any internet connection.
+* **Deterministic.** The same dependencies will be installed in the same exact way on any machine, regardless of install order.
+* **Network Performance.** Yarn efficiently queues up requests and avoids request waterfalls in order to maximize network utilization.
+* **Multiple Registries.** Install any package from either npm or Bower. Keep your package workflow the same.
```
Comment (thejameskyle) [inline README.md]: we should remove this bullet point

**Comment by Daniel15** [comment]
cc @JoelMarcey - Any other thoughts on these phrasing changes?

File: README.md [hunk 2]
```diff
@@ -19,23 +19,23 @@
 
 **Fast:** Yarn caches every package it downloads so it never needs to download the same package again. It also parallelizes operations to maximize resource utilization so install times are faster than ever.
 
-**Reliable:** Using a detailed, but concise, lockfile format, and a deterministic algorithm for installs, Yarn is able to guarantee that an install that worked on one system will work exactly the same way on any other system.
+**Reliable:** Thanks to a detailed (but concise) lockfile format and a deterministic algorithm for installs, Yarn is able to guarantee that an install that worked on one system will work exactly the same way on any other system.
```
Comment (Daniel15) [inline README.md]: What about 
```diff
-Using a detailed, but concise, lockfile format, and a deterministic algorithm for installs,
+Using a detailed, concise lockfile format and a deterministic algorithm for installs,
```

I don't think we need the "but".

File: README.md (hunk 1 above)
Comment (JoelMarcey) [inline README.md]: The change here seems reasonable, but if @thejameskyle thinks it should be removed, then let's remove it.

**Comment by JoelMarcey** [comment]
@Daniel15 The changes seem good and reasonable to me.

File: README.md (hunk 1 above)
Comment (thejameskyle) [inline README.md]: Bower support is being removed

**Pull request merged**